sympy/printing/ccode.py

```diff
@@ -195,7 +195,7 @@ def _print_Piecewise(self, expr):
             for i, (e, c) in enumerate(expr.args):
                 if i == 0:
                     lines.append("if (%s) {" % self._print(c))
-                elif i == len(expr.args) - 1 and c == True:
```

Comment:
Why did you make this change? I believe this changes the behavior of the function.